screens/request_profile.js

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image, FlatList} from 'react-native';
+
+const window = Dimensions.get("window")
+
+let colors = ['#6DC4E0', '#605DF1']
+
+let items = [
+  {
+    name: 'Item',
+    q: 'Q',
+    comment: 'Comments',
+    check: require('./../assets/images/check.png'),
+  },
+  {
+    name: 'Apple',
+    q: 3,
+    comment: 'Granny Smith',
+    check: require('./../assets/images/checkbox.png'),
+  },
+  {
+    name: 'Apple',
+    q: 3,
+    comment: 'Granny Smith',
+    check: require('./../assets/images/checkbox.png'),
+  },
+  {
+    name: 'Apple',
+    q: 3,
+    comment: 'Granny Smith',
+    check: require('./../assets/images/checkbox.png'),
+  }
+]
+
+export default class Request extends React.Component { 
+  render() {
+    return (
+    <View>
+          <View style={styles.container}>
+            <View style={styles.topBanner} />
```

Comment:
You should be using @sanjanagoli Toolbar component. Her PR needs to be merged before you can complete this task.